sdks/python/apache_beam/pipeline.py

```diff
@@ -620,23 +620,25 @@ def apply(self, transform, pvalueish=None, label=None):
         current.add_output(result, result._main_tag)
         continue
 
+      # TODO(BEAM-9322): Remove the experiment check and have this conditional
+      # be the default.
+      if self._options.view_as(DebugOptions).lookup_experiment(
+          'passthrough_pcollection_output_ids', default=False):
+        # Otherwise default to the new implementation which only auto-generates
```

Comment:
I think this comment could be improved, because this is not the otherwise case anymore. Follow up PR would be fine.